### DISCUSSION TITLE
Monogame integration!

### DISCUSSION THREAD

**Discussion by gaetandezeiraud** [SupinePandora43/UltralightNet#77] [Show and tell]
Hi!
Maybe it can interest someone, here a simple integration of UltralightNet with Monogame.
https://github.com/gaetandezeiraud/UltralightNet-monogame

![image](https://github.com/SupinePandora43/UltralightNet/assets/1573282/4f9d9015-2c75-4bd1-a7c2-9eea3b8195fd)

Enjoy!

**Comment by LiberRuth** [comment]
GIF is not played repeatedly
